src/app/components/HomePageNonCritical.tsx

```diff
@@ -28,6 +28,8 @@ import TechDocIllustration from '../../images/tech_doc_illustration.svg';
 import PlaceHolder from './PlaceHolder';
 import { Location, LocationToPath } from '../config/urls';
 
+import './styles/home-page-non-critical.scss';
```

Comment:
Should we slowly add new stylesheet files as CSS modules so that we get used to it?

Reply:
Ah yes, didn't know if that had gone in yet!